### DISCUSSION TITLE
Can I connect to my raspberry pi 4?

### DISCUSSION THREAD

**Discussion by TheFl1y** [innoveit/react-native-ble-manager#1009] [Q&A]
Is it possible to connect my raspberry pi 4 use react-native-ble-manager ?

**Comment by marcosinigaglia** [comment]
Hi @TheFl1y , I think is possible, try to follow this tutorial https://punchthrough.com/creating-a-ble-peripheral-with-bluez/

**Reply by lotalogic** [reply]
Funny this is the same example I have found to solve my issue on iOS ...

**Comment by lotalogic** [comment]
This is how I do it [https://github.com/lotalogic/debug-ble-server-ios](https://github.com/lotalogic/debug-ble-server-ios).
But for now only works with an android client
